wallet-new/src/Cardano/Wallet/Kernel/Internal.hs

```diff
@@ -50,6 +51,7 @@ data PassiveWallet = PassiveWallet {
       -- ^ An opaque handle to a place where we store the 'EncryptedSecretKey'.
     , _wallets          :: AcidState DB
       -- ^ Database handle
+    , _walletRocksDB    :: MonadDBReadAdaptor IO
```

Comment:
I thought part of the goal was to avoid `MonadDBRead` from leaking into the new `wallet-new` datatypes.

Reply:
The goal was to make `MonadDBRead` available in the kernel without making all the kernel types have all these constraints, and that's precisely what this achieves.